options/posix/generic/pthread-stubs.cpp

```diff
@@ -863,23 +925,25 @@ int pthread_cond_destroy(pthread_cond_t *) {
 }
 
 int pthread_cond_wait(pthread_cond_t *__restrict cond, pthread_mutex_t *__restrict mutex) {
+	struct timespec *time = nullptr;
```

Comment:
Don't think this is correct..

Reply:
Yeah you're right, I appear to have made this while still sleeping. I'll fix it asap.